src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -26,7 +28,12 @@ export default class MyProfile extends Component {
         showPerms: PropTypes.bool,
         showHomeLink: PropTypes.bool,
       }),
+      okapi: PropTypes.object,
     }).isRequired,
+    modules: PropTypes.shape({
+      app: PropTypes.array,
```

Comment:
Do we really need `app` propType specified in PropTypes?

Reply:
Sure we do, if we're accessing `props.modules.app`.

Reply:
(After reading on ...)

OK, I see that we do _not_ access `props.modules.app`. So you were right to question this, @VictorSoroka1.

Reply:
But ...

A better question would be _why_ we don't use `props.modules.app`. I think we should be extracting links from modules of type `app` as well as those of type `settings`. That entails a change to line 60, below.

Reply:
Makes sense.